xbmc/video/VideoDatabase.cpp

```diff
@@ -2072,6 +2072,8 @@ std::string CVideoDatabase::GetValueString(const CVideoInfoTag &details, int min
   std::vector<std::string> conditions;
   for (int i = min + 1; i < max; ++i)
   {
+    if (offsets[i].type == NULL || offsets[i].offset == NULL)
+      continue;
```

Comment:
No issues. With this check I can set the offset to null so I can drop 2 variables that we constantly fill and carry around even if they're not used in CVideoInfoTag.  
That's unrelated I'll split it in a separate commit